### ISSUE TITLE
Minimum preview scale factor

### ISSUE THREAD

**Issue by Mc-Zen** [typst/webapp-issues#114]
The current minimum preview scale factor in the web app is 25%. For large-scale documents like posters it would be beneficial to allow smaller values than that.

**Comment by reknih** [comment]
You are right, imposing a percentage limit is not great for large pages. Instead, we should limit the zoom factor with regard to the size of the pages: Surely, no one wants to zoom out so much that all pages in their document are smaller than 20x20 pixels.
